recipes/djvulibre/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "djvulibre" %}
+{% set version = "3.5.28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/djvu/DjVuLibre/{{ version }}/djvulibre-{{ version }}.tar.gz
+  sha256: fcd009ea7654fde5a83600eb80757bd3a76998e47d13c66b54c8db849f8f2edc
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - make
+  host:
+    - librsvg
+    - libtiff
+    - libjpeg-turbo
+    - jpeg
+  run:
+    - libjpeg-turbo
+    - libtiff
+    - jpeg
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/ddjvu
```

Is there a `--help`/`--version` or similar you can run?
```suggestion
    - ddjvu --help
```

---

Fixed in https://github.com/conda-forge/staged-recipes/pull/16277/commits/dae28305e64d5c81cce57463bf086b40765aae12